stub-idp/src/main/java/uk/gov/ida/stub/idp/resources/idp/LoginPageResource.java

```diff
@@ -108,6 +139,7 @@ public Response post(
             case SignIn:
                 try {
                     idpUserService.attachIdpUserToSession(idpName, username, password, sessionCookie);
+                    idpUser = allIdpsUserRepository.getUserForIdp(idpName, username);
```

Comment:
`idpUser` isn't used anywhere (as far as I can see) ... do we need it?

Reply:
Deleted.